kv/memberlist/memberlist_client.go

```diff
@@ -125,36 +125,36 @@ func (c *Client) awaitKVRunningOrStopping(ctx context.Context) error {
 type KVConfig struct {
 	// Memberlist options.
 	NodeName            string        `yaml:"node_name"`
```

Comment:
Even node name may be considered advanced, since it has a good default: hostname.